docs/user/types/branches/branch/category-usage.en.md

```diff
@@ -14,3 +14,4 @@ If the value is `product_version`, the property `name` must not contain a versio
     * `earlier`
     * `later`
     * `prior`
+    * `version`
```

Comment:
My bad - it is actually called `versions`. See https://docs.oasis-open.org/csaf/csaf/v2.0/cs02/csaf-v2.0-cs02.html#6131-version-range-in-product-version

Reply:
I didn't catch it either. Updated.